docs/guides/dynamo_deploy/dynamo_cloud.md

```
@@ -56,58 +53,20 @@ Just export the environment variable. This will be the image used by your indivi
-
-```bash
-earthly --push +all-docker --DOCKER_SERVER=$DOCKER_SERVER --IMAGE_TAG=$IMAGE_TAG
-```
-
-### 🚀 Deploying the Dynamo Cloud Platform
+## 🚀 Deploying the Dynamo Cloud Platform
 
-Once you've built and pushed the components, you can deploy the platform to your Kubernetes cluster.
-
-### Prerequisites
+## Prerequisites
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)